dc/s2n-quic-dc/src/packet/uds/decoder.rs

```diff
@@ -15,6 +15,7 @@ pub struct Packet {
     export_secret: Vec<u8>,
     application_params_version: u8,
     application_params: ApplicationParams,
+    encode_time: i64, // CLOCK_MONOTONIC_RAW in microseconds
```

Comment:
I'd expect this to be unsigned. If we use nix, we'd do something like `Duration::from(clock_gettime()).as_micros() as u64` to get this.